Source/WTF/wtf/PlatformHave.h

```diff
@@ -1301,3 +1301,9 @@
 #define HAVE_AVSAMPLEBUFFERDISPLAYLAYER_COPYDISPLAYEDPIXELBUFFER 1
 #endif
 #endif
+
+#if (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED < 160000) \
+    || PLATFORM(WATCHOS) \
```

Comment:
Little surprised WatchOS and TvOS don't have a version check. Is it really iOS specific?